### PR TITLE
ADS release 2.8.9

### PR DESCRIPTION
**Pull request by liudmylaru**
### Description

Running release v2.8.9. Jira: ODSC-45687

- added release notes
- bumped version to 2.8.9
- minor docs fix

### Pre-commit check
```
check python ast.....................................(no files to check)Skipped
check docstring is first.............................(no files to check)Skipped
check json...........................................(no files to check)Skipped
check for merge conflicts................................................Passed
check yaml...........................................(no files to check)Skipped
detect private key.......................................................Passed
fix end of files.........................................................Passed
pretty format json...................................(no files to check)Skipped
trim trailing whitespace.................................................Passed
black................................................(no files to check)Skipped
rst ``code`` is two backticks............................................Passed
rst ``inline code`` next to normal text..................................Passed
Detect hardcoded secrets.................................................Passed
check-copyright......................................(no files to check)Skipped
```

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Release Notes
 =============
 
+2.8.9
+-----
+Release date: August 30, 2023
+
+* Upgraded the ``scikit-learn`` dependency to ``>=1.0``.
+* Upgraded the ``pandas`` dependency to ``>1.2.1,<2.1``. This will allow users to use ADS with pandas 2.0.
```
Comment (lvallad) [inline docs/source/release_notes.rst:10]:
```suggestion
* Upgraded the ``pandas`` dependency to ``>1.2.1,<2.1`` to allow you to use ADS with pandas 2.0.
```

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+* Fixed "Attribute not found" error, when ``deploy()`` called twice in ``GenericModel``.
```
Comment (lvallad) [inline docs/source/release_notes.rst:11]:
```suggestion
* Fixed rhe "Attribute not found" error, when ``deploy()`` called twice in ``GenericModel``.
```

File: docs/source/release_notes.rst [hunk 3]
```diff
@@ -2,6 +2,15 @@
...
+* Fixed fetch of the security token, when relative path for the ``security_token_file`` provided (used in session token-bases authentication).
```
Comment (lvallad) [inline docs/source/release_notes.rst:12]:
```suggestion
* Fixed the fetch of the security token, when the relative path for the ``security_token_file`` is provided (used in session token-bases authentication).
```

File: docs/source/release_notes.rst (hunk 1 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: updated

File: docs/source/release_notes.rst (hunk 3 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: updated

File: docs/source/release_notes.rst (hunk 2 above)
Comment (liudmylaru) [reply docs/source/release_notes.rst]: updated

**Review by lvallad — APPROVED** [review]

**Review by qiuosier — DISMISSED** [review]

**Review by mrDzurb — DISMISSED** [review]

**Review by mrDzurb — DISMISSED** [review]

**Review by qiuosier — DISMISSED** [review]

**Review by qiuosier — APPROVED** [review]

**Review by mrDzurb — APPROVED** [review]

**Review by mrDzurb — APPROVED** [review]

**Review by mingkang111 — APPROVED** [review]

**Pull request merged**